pkg/controller/jobs/raycluster/raycluster_webhook.go

```diff
@@ -60,6 +61,12 @@ func (w *RayClusterWebhook) Default(ctx context.Context, obj runtime.Object) err
 	log := ctrl.LoggerFrom(ctx).WithName("raycluster-webhook")
 	log.V(10).Info("Applying defaults", "job", klog.KObj(job))
 
+	// Do not default suspend a RayCluster that's managed by a RayJob
+	if owner := metav1.GetControllerOf(job.Object()); owner != nil && jobframework.IsOwnerManagedByKueue(owner) {
```

Comment:
Isn't this a problem for jobs too?

Reply:
jobs that belong to jobsets, that is.

Reply:
In that case, can we move the logic inside `jobframework.ApplyDefaultForSuspend`?

Reply:
That's a good idea. It'll apply to a larger set of types that's what required in practice , but I don't see any issue with that.

Reply:
Updated, PTAL.